### DISCUSSION TITLE
What is the next number in the sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#514] [Galaxy Trivia] [answered]
2, 3, 5, 9, 17, _

**Comment by peterbaccaro** [comment, marked as answer]
The difference between the numbers in the sequence is 1, 2, 4 and 8 so the the increment is double the previous increment.  The next increment would therefore be 16, so the next number would be 17+16 which would be **33**.

**Comment by thuongtruong1009** [comment]
The next number in the sequence is 33.

The pattern in the sequence is obtained by adding successive odd numbers to the previous term starting with 1. For example:

2 + 1 = 3
3 + 2 = 5
5 + 4 = 9
9 + 8 = 17
17 + 16 = 33
Therefore, the next number is 33.
